utils/vcsdata.go

```diff
@@ -44,27 +48,25 @@ func (this *VcsDetails) get() int32 {
 	2. .git not found, go to parent dir and repeat
 	3. not found on the root directory, add all subpath to cache with nil as a value
 */
```

Comment:
Let's convert the comment of this function to use go style (// instead of /* */).